source/module_base/global_function.h

```diff
@@ -110,6 +110,37 @@ void AUTO_SET(const std::string &name,const T &a)
 //==========================================================
 void DONE(std::ofstream &ofs,const std::string &description, bool only_rank0 = false);
 
+
+//==========================================================
+// GLOBAL FUNCTION :
+// NAME : TASK_DIST_1D
+// Distributing 1d tasks by worker id
+//==========================================================
+template <typename T_task, typename T_out>
+static inline void TASK_DIST_1D(int nworker, int iworker, T_task ntask, T_out& start, T_out& len)
```

Comment:
yes, it is a very common fuction for parallelzation. it may be use on other parallel block, currently has been use in `src_pw/force.cpp`, `src_pw/stress_func_cc.cpp`, `src_pw/stress_func_ewa.cpp`

Reply:
Maybe it can be placed to another file in `module_base`